az-core/src/main/java/azkaban/Constants.java

```diff
@@ -693,6 +694,10 @@
     // Constant to enable pod for developer testing
     public static final String FLOW_PARAM_ENABLE_DEV_POD = "enable.dev.pod";
 
+    // Params with this prefix will be set as environment variables to the flow container and can
+    // be utilized by any script for download/installation.
+    public static final String FLOW_PARAM_RPM_PREFIX = "flow.container.rpm.";
```

Comment:
Can't we utilize pod.env.var. flow param? We are using it for Azkaban distribution already.